client/js/img/cards/card_1_9.svg

```diff
@@ -0,0 +1 @@
9.77,26.06,17.91,5.81,6.57,4.77,11-3.58,13.39C96,83.27,88.45,87.06,80.36,89.27a8.82,8.82,0,0,0-5.5,4.66c-4.57,8.2-8.12,17-14.16,24.35C58.75,120.66,57.86,121.44,55.59,121.47ZM50.91,31.52c-.29-.95-.62-1.94-1.89-2s-2.2.36-2.26,1.81c0,1.23.28,2.29,1.67,2.43S50.46,32.8,50.91,31.52Zm11,1.84c1.38,0,2-.71,2-2.09s-.59-2.06-1.8-2A2.19,2.19,0,0,0,60,31.47,1.71,1.71,0,0,0,61.92,33.36Z"/><path class="cls-5" d="M50.91,31.52c-.45,1.28-1,2.38-2.48,2.24s-1.72-1.2-1.67-2.43c.06-1.45,1-1.88,2.26-1.81S50.62,30.57,50.91,31.52Z"/><path class="cls-5" d="M61.91,33.36A1.71,1.71,0,0,1,60,31.47a2.19,2.19,0,0,1,2.15-2.19c1.21-.07,1.81.73,1.8,2S63.3,33.35,61.91,33.36Z"/><path class="cls-6" d="M60.09,105.29c-2.44.26-3.67-1.54-5.08-3.23-6-7.25-9.54-16-14-24.12a10.06,10.06,0,0,0-6.38-5.12C26.3,70.4,18.53,66.5,10.16,64.3a9.63,9.63,0,0,1-3.63-1.95,4.82,4.82,0,0,1-1.58-6c1.27-3.24,3.59-5.67,6-8A76.86,76.86,0,0,1,34,33.56c3.62-1.57,5.42-3.9,6.55-7.57C43.09,17.61,46.8,9.74,53.09,3.41c4.52-4.55,9.54-4.55,14.07,0,6.27,6.34,10,14.21,12.56,22.58,1.1,3.55,2.8,5.88,6.34,7.44,9.77,4.32,18.87,9.77,26.06,17.91,5.81,6.57,4.77,11-3.58,13.39-8.07,2.33-15.59,6.12-23.68,8.33a8.82,8.82,0,0,0-5.5,4.66c-4.57,8.2-8.12,17-14.16,24.35C63.25,104.48,62.36,105.26,60.09,105.29Z"/></symbol></defs><title>Classic</title><use width="180" height="252" xlink:href="#Card_layout_7"/><use width="25.28" height="26" transform="translate(14.99 46.17)" xlink:href="#Diamond"/><path class="cls-4" d="M29.1,30a13.4,13.4,0,0,1-2.79,1.51,7.13,7.13,0,0,1-2.56.49,6.15,6.15,0,0,1-5.14-2.67,11.28,11.28,0,0,1-2-7,14.44,14.44,0,0,1,2.81-9,8.29,8.29,0,0,1,6.77-3.7q4.21,0,6.82,4t2.62,10.7q0,8.06-3.72,13.4t-9,5.35a8.38,8.38,0,0,1-3.44-.68,7.65,7.65,0,0,1-2.7-2.09l5.11-5.82a10.14,10.14,0,0,0,1.94,2.65,3,3,0,0,0,2,.85,2.77,2.77,0,0,0,2.46-1.61,8.49,8.49,0,0,0,.94-4.32c0-.16,0-.43-.05-.82S29.14,30.53,29.1,30Zm-7-12.37a17,17,0,0,0,1.44,6.86c1,2.2,1.92,3.29,2.89,3.29a2,2,0,0,0,1.7-.81,4.36,4.36,0,0,0,.56-2.43,17.18,17.18,0,0,0-1.42-6.92q-1.43-3.29-3-3.28a1.8,1.8,0,0,0-1.6.84A4.46,4.46,0,0,0,22.06,17.65Z"/><use width="115.78" height="121.49" transform="translate(48.18 34.54) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(116.02 34.54) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(48.18 85.13) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(116.02 85.13) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(82.17 105.44) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(48.18 133.72) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(116.02 133.72) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(48.18 183.31) scale(0.43)" xlink:href="#Red_2"/><use width="115.78" height="121.49" transform="translate(116.02 183.31) scale(0.43)" xlink:href="#Red_2"/></svg>
```

Comment:
Not this car specifically but these all need to be optimized. They have `id`s and all sorts of other attributes we probably don't want. Not for this PR, but maybe after Vue 3 (and Vite) we look in to an SVG loader or a tool we can run to optimize them.